docs/how-tos/configure.md

```diff
@@ -77,25 +73,33 @@ Click on the **CONFIGURE** tab of your machine's page in the Viam app to navigat
 {{% tablestep link="/configure/#components" %}}
 **4. Configure your components**
 
-Each physical piece of your smart machine that is controlled by a computer is called a {{% glossary_tooltip term_id="component" text="_component_" %}}. For example, if your smart machine includes an arm, a motor, and a camera, each of those is a component.
+You will now create a configuration that describes any connected hardware as well as any software your device uses.
+The {{% glossary_tooltip term_id="component" text="_components_" %}} in your configuration can describe:
+
+- Physical hardware, such as arms, motors, or cameras.
+- Wrappers that provide additional functionality for other configured components, for example a failover motor that switches from one motor to another if the primary motor fails.
+- Software that maps to the API calls that you would use for existing hardware types. For example, a temperature sensor may provide temperature readings based on an online service instead of hardware readings. Another such type of sensor that is frequently used returns _readings_ confirming it is safe to sync data or reconfigure a machine.
+- Software that does not map well to the existing APIs, such as a switch that you'd like to be able to turn on an off. For these types of components, you use the generic component API.
```

Comment:
@IanWhalen I am rewriting this based on yesterday's discussion to see how we can trim/clarify this. While I've trimmed the steps, I think there is actually vital information missing. The physical hardware part is fairly obvious. The other three aren't. Especially the latter two are things that once people get it, get folks excited - based on our SEs.

We need to also consider teachable moments as we shorten content. At some point we do need to explain what's possible with the platform. I think we can skip over other parts of this but I'd assume this isn't one. Something to run by the SEs though.

Reply:
i don't know if I want to think about this in terms of a diff.  I expect I would like everything to be written from scratch as much as possible given the new IA we come up with.

Reply:
ok - I will merge this for now, since it removes some weirdness and we can see how a rewrite turns out